### PR TITLE
Footer Quiz Implementation

### PR DESCRIPTION
**Pull request by molaycule**
Finish the Fill In The Blank quiz (2nd to last section of About the Amazon)

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -72,6 +72,27 @@ function dashedBordersPlugin({ matchUtilities, theme }: PluginAPI) {
   );
 }
 
+function dashedBordersAccentPlugin({ matchUtilities, theme }: PluginAPI) {
```
Comment (Kokanm) [inline tailwind.config.ts:75]: Why was this needed?

File: src/components/content/activities/fill-in-the-blank.tsx [hunk 2]
```diff
@@ -305,14 +351,23 @@ export function FillInTheBlankActivity({
...
 
           {props.subText && (
-            <p className="mt-2 text-4xl leading-snug text-neutral-dark-700">
+            <p
+              className={clsx(
+                "mt-2 text-4xl leading-snug",
+                props.textColorStyle === "dark" && "text-neutral-dark-700",
+                props.textColorStyle === "light" && "text-neutral-100",
+                props.textColorStyle === "primary" && "text-primary-100",
+              )}
```
Comment (Kokanm) [inline src/components/content/activities/fill-in-the-blank.tsx:369]: This is repeated several times, maybe you can make a variable `textColor` that is assigned in a **switch** statement and then just use it everywhere in the file.
```suggestion
              className={clsx(
                "mt-2 text-4xl leading-snug",
                textColor,
              )}
```

File: tailwind.config.ts (hunk 1 above)
Comment (molaycule) [reply tailwind.config.ts:75]: It was for the dashed border color for the Fill in the option
image

File: src/components/content/activities/fill-in-the-blank.tsx (hunk 2 above)
Comment (molaycule) [reply src/components/content/activities/fill-in-the-blank.tsx]: I have push a refactor commit to address this

**Review by Kokanm — APPROVED** [review]
